Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/Plugins/StandardPlugins/Source/MotionWindow/MotionWindowPlugin.cpp

```diff
@@ -120,21 +121,22 @@ namespace EMStudio
     MotionWindowPlugin::MotionWindowPlugin()
         : EMStudio::DockWidgetPlugin()
     {
-        m_dialogStack                        = nullptr;
         m_motionListWindow                   = nullptr;
         m_motionPropertiesWindow             = nullptr;
-        m_motionExtractionWindow             = nullptr;
-        m_motionRetargetingWindow            = nullptr;
         m_dirtyFilesCallback                 = nullptr;
         m_addMotionsAction                   = nullptr;
         m_saveAction                         = nullptr;
-        m_motionNameLabel                    = nullptr;
     }
 
 
     MotionWindowPlugin::~MotionWindowPlugin()
     {
-        delete m_dialogStack;
+        if (m_motionPropertiesWindow)
```

Comment:
no need to have a condition before deleting, delete call handles using `nullptr`.